packages/theme/helpers/checkout.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const getShippingMethodPrice = (shippingMethod, total: number): number => {
```

Comment:
Do you use it? Rather it's commercetools related

Reply:
For now I just copy/pasted it but it will be used later when developed this feature -> https://www.vendure.io/docs/graphql-api/shop/queries/#eligibleshippingmethods. I will create an issue to convert this method to appriopriate in Vendure